src/components/CartoonsList.jsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from 'react';
+// import CartoonsInfo from './CartoonsInfo'
+import CartoonCards from './CartoonCards'
+
+export default function CartoonsList (){
+    const initialList = [
+        {
+            id:1,
+            name:'Remi the Homeless Girl',
+            favorite: false,
+            img: require('../assets/images/remi.jpg')
```

Comment:
try to import the image using Import instead of require

Reply:
done.